<launch>


    <param name="/pkg_path/vision" value="$(find vision)" />
    <param name="/pt_path" value="/scripts/v5l-last.pt" />
    <node name = "vision_node" pkg = "vision" type="vision_main.py" output = "screen"/>
    <!--
    <node name = "test_vision_node" pkg = "vision" type="testVis.py" output = "screen"/>
    -->

</launch>
